changelog/unreleased/features/6833.md

```diff
@@ -0,0 +1,3 @@
+- Added `RoadComponent.language` value.
+- Changed `EHorizonEdgeMetadata.names` class from `RoadName` to `RoadComponent`.
+- Removed `RoadName` class.
```

Comment:
This entry isn't needed.

Reply:
Sorry, I thought we _replaced_ `RoadName` with `RoadComponent` but that wasn't the case. Let's leave the entry and add migration instructions as mentioned below.